### ISSUE TITLE
No route matches [POST] "/xxxx/3/edit" 

### ISSUE THREAD

**Issue by charly-palencia** [sferik/rails_admin#2293]
## Description
When i tried to edit some of my products or categories i got a POST RoutingError. I expect it sends the request to the  PUT action but it doesn't. what do i miss? PATH action probably? i'm not sure.
```
Started POST "/admin/product/3/edit" for ::1 at 2015-05-09 10:35:43 -0500

ActionController::RoutingError (No route matches [POST] "/admin/product/3/edit"):
  actionpack (4.2.0) lib/action_dispatch/middleware/debug_exceptions.rb:21:in `call'
  actionpack (4.2.0) lib/action_dispatch/middleware/show_exceptions.rb:30:in `call'
  railties (4.2.0) lib/rails/rack/logger.rb:38:in `call_app'
```

### Versions
**Rails** 4.2
**Rails_admin** 0.6.7

## router.rb
```ruby
  root :to => redirect('/admin')
  namespace :api do
    namespace :v1 do
      resources :categories
      resources :products
    end
  end

  mount RailsAdmin::Engine => '/admin', as: 'rails_admin'
```

### initializer/rails_admin.rb

```ruby

  config.actions do
    dashboard                     # mandatory
    index                         # mandatory
    new
    export
    bulk_delete
    show
    edit
    delete
    show_in_app
```

**Comment by mshibuya** [comment]
Can you create minimal app to reproduce this problem?

**Comment by charly-palencia** [comment]
@mshibuya  thanks for your replay. I fixed my issues while i was creating my new app :)
 
Apparently my problem was about a conflict with rails-api. I tried with [this](http://stackoverflow.com/questions/10923280/rails-admin-with-rails-api-conflict) but it doesn't work. So, i figured it out with this:

Gemfile
```
gem 'rails-api', require: false 
```

api_base_controller file

```
require 'rails-api'
```

Thanks you so much!
